pep-0690.rst

```diff
@@ -193,6 +193,17 @@ a list of module names within which all imports will be eager::
 The effect of this is also shallow: all imports within ``one.mod`` will be
 eager, but not imports in all modules imported by ``one.mod``.
 
+``set_eager_imports()`` can also take a callback which receives a module name and returns
+whether the import of this module should be eager::
```

Comment:
We just realized in discussion that this is subtly wrong, should be fixed like so:

```suggestion
whether imports within this module should be eager::
```